ui-tests/__test__/main.spec.ts

```diff
@@ -16,7 +16,7 @@ describe('Open MynahUI', () => {
   beforeAll(async () => {
     const browserName = await (await browser.browserType()).name();
     const toMatchImageSnapshot = configureToMatchImageSnapshot({
-      failureThreshold: 0.12,
+      failureThreshold: 0.1,
```

Comment:
Wasn't 0.12 the perfect value for this, why was it changed to 0.1?

Reply:
Fixing the input field border, which was causing minimal movements that was causing errors.